modules/ROOT/pages/configure-local-registry.adoc

```diff
@@ -124,3 +124,98 @@ This command ensures that Runtime Fabric creates the necessary overrides for the
+
+. In your shell script, run the following script to synchronize container images to a local registry. Note that you must update `PLATFORM_URL`, `AGENT_VERSION`,  `AUTHORIZATION_TOKEN`, and `ACTIVATION_DATA`.
++
+The following script is an example:
++
+[source,copy]
+----
+##!/usr/bin/env bash
+ 
+PLATFORM_URL=<platform-url>
+AGENT_VERSION=<agent-version>
+AUTHORIZATION_TOKEN=<authorization-token>
+MULESOFT_DOCKER_SERVER=rtf-runtime-registry.kqa.msap.io
+PRIVATE_REGISTRY_LOCATION=us-west2-docker.pkg.dev/mulesoft-rtf-gke-test
+ACTIVATION_DATA=xxxx
+#MULE_RUNTIME=poseidon-runtime-4.4.0:20231211-2
+MULE_RUNTIME=poseidon-runtime-4.6.0:33e-java8 
+ 
+echo $PLATFORM_URL ; echo $AGENT_VERSION ; echo $MULESOFT_DOCKER_SERVER ; echo $PRIVATE_REGISTRY_LOCATION
+eval $(rtfctl get registry-credentials ${ACTIVATION_DATA})
+docker login $RTF_IMAGE_REGISTRY_ENDPOINT --username $RTF_IMAGE_REGISTRY_USER --password $RTF_IMAGE_REGISTRY_PASSWORD
+ 
+dependencies=$(curl "https://${PLATFORM_URL}/runtimefabric/api/agentmanifests/${AGENT_VERSION}" \
+-H "Authorization: bearer $AUTHORIZATION_TOKEN" | jq -c '.dependencies |.[] | select(.provider | contains("generic")) | "\(.artifact):\(.version)"' | sed 's/"//g')
+echo ${dependencies}
+for i in $dependencies
+do
+   echo "Processing $i"
+   docker pull "${MULESOFT_DOCKER_SERVER}/mulesoft/${i}"
+   docker tag  "${MULESOFT_DOCKER_SERVER}/mulesoft/${i}" "${PRIVATE_REGISTRY_LOCATION}/mulesoft/${i}"
+   docker push "${PRIVATE_REGISTRY_LOCATION}/mulesoft/${i}"
+done
+ 
+docker pull "${MULESOFT_DOCKER_SERVER}/mulesoft/${MULE_RUNTIME}"
+docker tag  "${MULESOFT_DOCKER_SERVER}/mulesoft/${MULE_RUNTIME}" "${PRIVATE_REGISTRY_LOCATION}/mulesoft/${MULE_RUNTIME}"
+docker push "${PRIVATE_REGISTRY_LOCATION}/mulesoft/${MULE_RUNTIME}"
+----
+
+[start=2]
+. Create a Runtime Fabric using Runtime Manager and select your desired Kubernetes service.
+. In Runtime Manager, select the *Helm* path, and follow the instructions on the screen platform.
+. Obtain your private `docker-server`, `docker-username`, and `docker-password`.
+. Following the *Helm* path instructions in Runtime Manager, create a secret with your docker server name, username, and password by running the following command in your Unix shell:
++
+[source,copy]
+----
+kubectl create secret docker-registry rtf-pull-secret --namespace test-gke-rtf-qax-helm --docker-server=<docker-servername> --docker-username=<docker-username --docker-password=<docker-password> --docker-email=<docker-email>
+----
+
+[start=6]
+. Following the *Helm* path instructions, download the values.yaml file and modify its values. The following is a values.yaml file example:
++
+[source,copy]
+----
+activationData: xxx
+proxy:
+ http_proxy:
+ http_no_proxy:
+ monitoring_proxy:
+muleLicense: 
+customLog4jEnabled: false
+global:
+ crds:
+  install: false
+ authorizedNamespaces: false
+ image:
+  rtfRegistry: us-west2-docker.pkg.dev/mulesoft-rtf-gke-test
```

Comment:
We should tell customers to put their own registry here and not make reference to out specific example.

Reply:
you may want to remove all specific values, ie.
PLATFORM_URL=<platform-url>
AGENT_VERSION=<agent-version>
AUTHORIZATION_TOKEN=<authorization-token>
MULESOFT_DOCKER_SERVER=rtf-runtime-registry.kprod.msap.io
PRIVATE_REGISTRY_LOCATION=<private-registry-location>
ACTIVATION_DATA=<rtf-activation-data>
MULE_RUNTIME=<mule-runtime-version-that-you-want>